src/script/script.h

```diff
@@ -68,144 +68,153 @@ std::vector<unsigned char> ToByteVector(const T& in)
+    OP_NIP = 0x77,          // remove second from top stack item
+    OP_OVER = 0x78,         // copy second from top stack item to the top
+    OP_PICK = 0x79,         // copy item N back in stack to the top
+    OP_ROLL = 0x7a,         // move item N back in stack
+    OP_ROT = 0x7b,          // move third from top stack item to the top
+    OP_SWAP = 0x7c,         // swap top two items of stack
+    OP_TUCK = 0x7d,         // copy top stack item and insert before second from top item
 
     // splice ops
-    OP_CAT = 0x7e,
-    OP_SUBSTR = 0x7f,
-    OP_LEFT = 0x80,
-    OP_RIGHT = 0x81,
-    OP_SIZE = 0x82,
+    OP_CAT = 0x7e,    // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_SUBSTR = 0x7f, // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_LEFT = 0x80,   // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_RIGHT = 0x81,  // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_SIZE = 0x82,   // push the length of top stack item (not pop the top element whose size is inspected)
 
     // bit logic
-    OP_INVERT = 0x83,
-    OP_AND = 0x84,
-    OP_OR = 0x85,
-    OP_XOR = 0x86,
-    OP_EQUAL = 0x87,
-    OP_EQUALVERIFY = 0x88,
-    OP_RESERVED1 = 0x89,
-    OP_RESERVED2 = 0x8a,
+    OP_INVERT = 0x83,      // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_AND = 0x84,         // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_OR = 0x85,          // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_XOR = 0x86,         // disabled, fail the script unconditionally (apply in an unexecuted conditional branch)
+    OP_EQUAL = 0x87,       // return 1 if the inputs are equal, 0 otherwise
+    OP_EQUALVERIFY = 0x88, // execute OP_EQUAL, then OP_VERIFY afterward
+    OP_RESERVED1 = 0x89,   // mark tx invalid unless occuring in an unexecuted OP_IF branch
+    OP_RESERVED2 = 0x8a,   // mark tx invalid unless occuring in an unexecuted OP_IF branch
 
     // numeric
-    OP_1ADD = 0x8b,
-    OP_1SUB = 0x8c,
-    OP_2MUL = 0x8d,
-    OP_2DIV = 0x8e,
-    OP_NEGATE = 0x8f,
-    OP_ABS = 0x90,
-    OP_NOT = 0x91,
-    OP_0NOTEQUAL = 0x92,
-
-    OP_ADD = 0x93,
-    OP_SUB = 0x94,
-    OP_MUL = 0x95,
-    OP_DIV = 0x96,
-    OP_MOD = 0x97,
-    OP_LSHIFT = 0x98,
-    OP_RSHIFT = 0x99,
-
-    OP_BOOLAND = 0x9a,
-    OP_BOOLOR = 0x9b,
-    OP_NUMEQUAL = 0x9c,
-    OP_NUMEQUALVERIFY = 0x9d,
-    OP_NUMNOTEQUAL = 0x9e,
-    OP_LESSTHAN = 0x9f,
-    OP_GREATERTHAN = 0xa0,
-    OP_LESSTHANOREQUAL = 0xa1,
-    OP_GREATERTHANOREQUAL = 0xa2,
-    OP_MIN = 0xa3,
-    OP_MAX = 0xa4,
-
-    OP_WITHIN = 0xa5,
+    OP_1ADD = 0x8b,      // 1 added to the top stack item
+    OP_1SUB = 0x8c,      // 1 subtracted from the top stack item
```

Comment:
would also use the active form here
```suggestion
    OP_1ADD = 0x8b,      // add 1 to the top stack item
    OP_1SUB = 0x8c,      // subtract 1 from the top stack item
```